src/index.html

```diff
@@ -16,7 +16,7 @@
         <a href="https://discord.gg/6qZ2CR4">
           <img class="mx-auto mb-2" src="/assets/logo.png" />
         </a>
-        <section class="flex flex-row-reverse justify-center">
+        <section class="flex flex-row-reverse justify-center hidden lg:flex">
           <div class="fly-in hidden lg:block">
```

Comment:
```suggestion
          <div class="fly-in">
```
Given the parent container will be hidden on mobile, no need to hide the children too

Reply:
good point i'll change that